Assets/Scripts/BulletCollisions.cs

```diff
@@ -0,0 +1,10 @@
+using UnityEngine;
+
+public class BulletCollisions : MonoBehaviour
+{
+    void OnCollisionEnter2D(Collision2D collision)
+    {
+        //Damage enemy here once
+        Destroy(gameObject);
```

Comment:
Based on this comment https://github.com/karoluzas/gamedev-project/pull/19#discussion_r999356126
as I understand this "Destroy(gameObject);" is actually destroying the bullet, not damaging the enemy right?

Reply:
Yes, as there are no enemies yet, damaging them will also be implemented later.

Reply:
Maybe then change comment to "//TODO - Damaging the enemy here once" that it wouldn't be misleading, I mean for me personally at first glance this Destroy sounded like it meant "Damage the enemy".